install/tests/disable-prompt.sh

```diff
@@ -15,7 +15,13 @@ cleanup () {
 }
 trap cleanup EXIT
 
-./swiftly-install.sh -y
+if has_command apt-get ; then
+    apt-get remove -y zlib1g-dev
+elif has_command yum ; then
+    yum remove -y libcurl-devel
```

Comment:
These are just arbitrary packages from the dependency lists used to check if the whole set have been installed or not.